libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -12,6 +15,22 @@
       <div class="reading-list-item--details--author">
         {{ b.authors.join(',') }}
       </div>
+      <div *ngIf="b.finishedDate">
+        Finish Date :<strong>{{ b.finishedDate }}</strong>
+      </div>
+    </div>
+    <div>
+      <button
+        id="{{ 'markasFinish_' + i }}"
+        mat-icon-button
+        [color]="b.finishedDate ? 'warn' : 'success'"
+        [disabled]="b.finishedDate"
+        [attr.aria-label]="'Mark ' + b.title + ' As Finshed'"
+        (click)="finishReadingList(b)"
+        title="Mark as Finished"
+      >
+        <mat-icon>done</mat-icon>
+      </button>
```

Comment:
Updating HTML for mark book as Finished. Buttons for Success to print timestamp and marked as finished